lib/cocoapods/command.rb

```diff
@@ -121,5 +122,25 @@ def verify_lockfile_exists!
         raise Informative, "No `Podfile.lock' found in the project directory, run `pod install'."
       end
     end
+
+    def self.verify_xcode_license_approved!
+      unless xcode_license_approved?
+        UI.puts xcode_license_message.red
```

Comment:
Yeah,  I prefer @CalQL8ed-K-OS it's not an assertion where someone needs the additional stack trace